Resources/css/site.css

```diff
@@ -0,0 +1,148 @@
+.heading {
+    background-color: black;
+    padding: 20px;
+    height: 20px;
+    display: flex;
+    overflow: hidden;
+    align-items: center;
+}
+
+body {
+    margin: auto;
+    background-image: none;
+}
+.add-money {
+    background-image: url("../../piles-of-big-money.jpeg");
+}
+
+.invert {
+    transform: rotate(180deg);
+}
+
+button {
+    min-height: 40px;
+    background: rgb(2, 0, 36);
+    background: linear-gradient(
+        90deg,
+        rgba(2, 0, 36, 1) 0%,
+        rgba(9, 121, 111, 1) 0%,
+        rgba(0, 212, 255, 1) 100%
+    );
+    color: white;
+    border-radius: 4px;
+}
+
+.joke-button {
+    border-radius: 0px;
+}
+
+.api-button {
+    background: #d1d0d0;
+    color: black;
+}
+
+button:hover {
+    background: rgb(0, 0, 0);
```

Comment:
```suggestion
    background: rgb(0, 0, 0);
    color: white;
```
Just a design suggestion, as your Joke API button is currently black on black:
![image](https://user-images.githubusercontent.com/43832670/159825666-c3aa9d0d-4382-4f27-9e10-94515f0c1f05.png)